src/capabilities.js

```diff
@@ -70,7 +76,8 @@ export const NO_ROLE_CAPABILITIES = {
       { readOwn: true, writeOwn: true, readOthers: false, writeOthers: false },
     ]
   }),
-  roleAssignment: [],
+  // TODO: Does this make sense?
+  roleAssignment: [LEFT_ROLE_ID],
```

Comment:
The more I'm looking at this, the more it seems that this change isn't necessary (i.e. NO_ROLE being able to assign LEFT), but would like confirmation before undoing